djaoapp/templates/saas/_plan_short.html

```diff
@@ -1,57 +1,69 @@
-<!-- We assume that either the monthly recuring amount will
-be positive and/or the setup fee will. -->
+<!-- 
+  We assume that either the monthly recurring 
+  amount will be positive and/or the setup fee will.
+-->
+
 <div id="{{plan.slug}}" class="col-sm-6 col-md-4 col-lg-4 plan" data-plan="{{plan.slug}}">
-  <div class="plan-short">
+  <div class="plan-short mt-4 pb-2">
     <div class="plan-main">
-      <div class="text-center">
+        
+      <!-- Plan Title -->
+      <div class="text-center py-3">
         {% if plan.created_at|date_in_future %}
-        <p>{% trans %}Avalaible soon!{% endtrans %}</p>
+          <p>{% trans %}Available soon!{% endtrans %}</p>
         {% endif %}
-        <div class="page-header">
-          <h2 class="editable-plan" data-key="title">{{plan.title}}</h2>
+
+        <div class="page-header pt-3">
+          <h5 class="editable-plan font-weight-light" data-key="title">
+            {{plan.title}}
+          </h5>
         </div>
       </div>
+
+      <!-- Plan Description -->
       <div class="plan-test">
         <div class="plan-pricing">
-          <h2 class="text-center editable-plan edit-currency" data-key="period_amount">
+          <h1 class="text-center editable-plan edit-currency mb-0" data-key="period_amount">
```

Comment:
`<h1>` tags should be reserved for a single page title per [SEO guidelines](https://www.searchenginejournal.com/how-important-is-h1-tag-for-seo/261547). Use `<h2>` here and update the CSS accordingly.